privacyidea/lib/tokens/hotptoken.py

```diff
@@ -832,10 +832,11 @@ def enroll_via_validate(cls, g, content, user_obj):
         # Create a challenge!
         c = token_obj.create_challenge()
         # get details of token
-        init_details = token_obj.get_init_detail()
+        init_details = token_obj.get_init_detail(user=user_obj)
         detail["transaction_ids"] = [c[2]]
         chal = {"transaction_id": c[2],
-                "image": init_details.get("otpkey").get("img"),
+                "image": init_details.get("googleurl").get("img"),
+                "token_uri": init_details.get("googleurl").get("value"),
```

Comment:
I would remove this. This is currently not documented and defined.
I would like to avoid using adding an undocumented feature here.

Also there are no defined cases, where this text represantation can actually be used.

Reply:
Done in 0fe459c1